### PR TITLE
Misc cleanup

### PR DESCRIPTION
**Pull request by Shane32**
Remaining after #3741

### CODE REVIEW

File: src/GraphQL.DataLoader.Tests/GraphQLBuilderTests.cs [hunk 1]
```diff
@@ -20,20 +21,25 @@ public void AddDataLoader()
...
+        var options = new ExecutionOptions()
+        {
+            RequestServices = mockServiceProvider.Object
+        };
+        foreach (var action in actions)
+        {
+            await action.ExecuteAsync(options, _ => Task.FromResult<ExecutionResult>(null!));
+        }
+        options.Listeners.ShouldContain(instance);
+
```
Comment (gao-artur) [inline src/GraphQL.DataLoader.Tests/GraphQLBuilderTests.cs:42]: That's smart!

**Comment by gao-artur** [comment]
> @gao-artur please review and confirm I have this right :)

Yep, you got it right

File: src/GraphQL.DataLoader.Tests/GraphQLBuilderTests.cs [hunk 2]
```diff
@@ -1,3 +1,4 @@
+using System;
```
Comment (Shane32) [inline src/GraphQL.DataLoader.Tests/GraphQLBuilderTests.cs:1]:
```suggestion
```

File: src/GraphQL.Tests/DI/GraphQLBuilderBaseTests.cs [hunk 3]
```diff
@@ -1,3 +1,4 @@
+using System;
```
Comment (Shane32) [inline src/GraphQL.Tests/DI/GraphQLBuilderBaseTests.cs:1]:
```suggestion
```

**Pull request merged**